Client/Frontend/Browser/LocalRequestHelper.swift

```diff
@@ -14,12 +14,13 @@ class LocalRequestHelper: TabContentScript {
         guard let params = message.body as? [String: String], message.frameInfo.request.url?.isLocal ?? false else {
             return
         }
+        
+        guard let token = params["securitytoken"], token == UserScriptManager.messageHandlerToken.uuidString else {
+                    print("Missing required security token.")
+                    return
+        }
 
-        if params["type"] == "load",
```

Comment:
Initially I removed it since we don't need use the `param["type"] = "load"`, but there might be unintended consequences so adding it back. We can remove it in a follow-up PR.